lib/Hal.js

```diff
@@ -0,0 +1,67 @@
+
+class Hal {
+  constructor(phrase) {
```

Comment:
I really don't this this needs to be a class. Classes exist for when we need to maintain state specific to a set of functions. But your listen function overrides this state with each invocation. If you just made Rojo into a module of pure functions, you'd have the same results with less opportunity for bugs.